api/millenium_patron.py

```diff
@@ -206,15 +261,54 @@ def remote_patron_lookup(self, patron_or_patrondata):
 
     def _remote_patron_lookup(self, identifier):
         """Look up patron information for the given identifier."""
-        path = "%(barcode)s/dump" % dict(barcode=identifier)
-        url = self.root + path
-        response = self.request(url)
-        return self.patron_dump_to_patrondata(identifier, response.content)
-        
+        raw_record = self._remote_patron_lookup_raw(identifier)
+        return self.patron_dump_to_patrondata(identifier, raw_record)
```

Comment:
I was thinking about doing this as well, then passing around `patrondata`, but since the `patrondata` object is shared by all the different authentication schemes, I wasn't sure if it was a good idea to add millennium specific data to it. I was considering adding something to it like `ils_data` where we could store ILS specific data on the patron record. 

What do you think? I can change this up to store the data on the patron record if it makes more sense.

Reply:
That's a good point. I think the `ils_data` idea would work, or you could make a `MilleniumPatronData` subclass of `PatronData` that has the one extra thing.